update rate limit metrics (#11127)
istio/istio.io: content/en/docs/tasks/policy-enforcement/rate-limit/index.md

is configured to allow 10 requests/min. The filter is also configured to add an `x-local-rate-limit`
response header to requests that are blocked.

{{< tip >}}

> The statistics mentioned [here](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter#statistics) are disabled by default, you can enabled those with following annotations in deployment:

Reviewer: ```suggestion
The statistics mentioned [here](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter#statistics) are disabled by default. You can enabled them with the following annotations during deployment:
```

Author: updated

> The statistics mentioned [here](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter#statistics) are disabled by default. You can enabled them with the following annotations during deployment:

Reviewer: I'm confused by the word `here`, especially since you don't really mean `on this page`. This should be updated to something that describes the link. Something like mentioned on the [Envoy rating limiting page] might be better.

Author: updated the content.

> The statistics mentioned on the [Envoy rating limiting page](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter#statistics) are disabled by default. You can enable them with the following annotations during deployment:

Reviewer 2: can you clarify this is pod annotation?

> The statistics mentioned on the [Envoy rating limiting page](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter#statistics) are disabled by default. You can enabled them with the following annotations during deployment:

Reviewer: ```suggestion
The statistics mentioned on the [Envoy rating limiting page](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter#statistics) are disabled by default. You can enable them with the following annotations during deployment:
```
